### DISCUSSION TITLE
How to delete filed issue at certain github repository.

### DISCUSSION THREAD

**Discussion by Ali-Milani-13** [community/community#125447] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello. 

recently I filed an issue to the `teams-toolkit` of the Microsoft Devs. it was done completely by accident. I removed the body of the issue and changed its title to a dot, but I really need to remove that issue. 

what should I do?

**Comment by kazmatw** [comment, marked as answer]
Hello,

Accidentally filing an issue on GitHub can happen, but unfortunately, GitHub does not allow users to delete issues outright—only repository owners and collaborators with the necessary permissions can do this. Here's what you can do next:

Comment on the Issue: Politely explain the situation in a comment on the issue itself. Mention that it was created accidentally and request the repository maintainers to close or delete it. This is often the fastest way to alert the maintainers about the mistake.

Contact the Maintainers: If the repository has contact information for the maintainers (like an email in the README or a CONTRIBUTING guide), consider reaching out directly to explain your situation and ask for their assistance in removing the issue.

Close the Issue: While you cannot delete the issue yourself, you can close it. This won't remove the issue from the repository, but it will indicate to the maintainers and other users that it should no longer be considered active. You've already renamed it, so closing it can further minimize attention to it.

If you find this information useful, please consider selecting this as the answer to help others in the community identify effective solutions. Thank you!

**Comment by davevad93** [comment]
Hi @Ali-Milani-13 ,

It's only possible to delete an `issue` if you own the repository where you opened it or in the case of an organization's repo you must have `admin` or `owner` permissions, otherwise you can't do it unfortunately.
